sympy/printing/latex.py

```diff
@@ -1744,8 +1756,15 @@ def _print_Tuple(self, expr):
         return self._print_tuple(expr)
 
     def _print_list(self, expr):
-        return r"\left[ %s\right]" % \
-            r", \  ".join([self._print(i) for i in expr])
+        if self._settings['decimal_separator'] == 'comma':
+            return r"\left[ %s\right]" % \
+                r"; \  ".join([self._print(i) for i in expr])
+        elif self._settings['decimal_separator'] == 'period':
+            return r"\left[ %s\right]" % \
+                r", \  ".join([self._print(i) for i in expr])
+        else:
+            raise ValueError('DecimalSeparatorNotFoundError')
```

Comment:
Better to write a "proper" error message, e
g. "Unknown decimal separator." The error is ValueError, so no need to come up with an earth-like error message.

(Didn't notice this earlier.)

Reply:
So something like:
`raise ValueError("UnknownDecimalSeparator")` ?

Reply:
like 
```python
raise ValueError(
    'Unrecognized name for decimal separator: "%s"' %
    self._settings['decimal_separator'])`
```